tests/integration/testdata/package/aws-serverless-inline.yaml

```diff
@@ -12,9 +12,3 @@ Resources:
           return {'body': 'Hello World 2!', 'statusCode': 200}
       Handler: index.handler
       Runtime: python3.7
-      Events:
```

Comment:
Without API gateway how are we validating if Lambda is working? Are we invoking it directly?

Reply:
That brings up a good point, what we check for here is that we can deploy a particular template and check for return codes, but we dont attempt to check if the components that are deployed actually work. This may be a new set of end to end tests we may need to adopt.

Reply:
while it could be valuable to test the function, it’s not the scope of these tests. We want to make sure the deploy can happen successfully. I can see value in verifying configuration of what was deployed, but that kind of a grey area to me and not really testing lambda either.